public-api/src/main/java/org/pmiops/workbench/cdr/model/DbDomain.java

```diff
@@ -120,6 +121,20 @@ public DbDomain countValue(Long countValue) {
         return this;
     }
 
+    @Transient
+    public Long getParticipantCount() {
+        return participantCount;
+    }
+
+    public void setParticipantCount(Long participantCount) {
```

Comment:
These aren't used anywhere?

Reply:
Added it to the dbdomain model. They are not used anywhere for now but would be.